HelpSource/Classes/Event.schelp

```diff
@@ -36,31 +36,91 @@ Returns an empty event with link::#.defaultParentEvent:: as parent.
 method::silent
```

Comment:
+ "related" should include Classes/Environment and Classes/Dictionary
+ the short description in line 18 should read:

An Event specifies an action to be taken in response to a link::#-play:: message. Its key/value pairs specify the parameters of that action. Event inherits most of its methods from its superclasses, especially from link::Classes/Dictionary::. __+ Something explaining the usage and differences of code::parent:: and code::proto::.__